### ISSUE TITLE
"Sing a song" query reports issue

### ISSUE THREAD

**Issue by Fleker** [googlesamples/assistant-sdk-python#257]
@salmanfarisvp:

> @Fleker  Yes , if i ask 

> > "Sing a song"

> it's return 

> > "i don't know how to help with that 

> "
> i using google assistant python SDK on Raspberry pi 3, 

The report suggests it's happening for their account, as it affects both the pushtotalk and hotword samples. I am unable to reproduce.

@salmanfarisvp, which spoken language are you using to make these queries?

**Comment by shavanni** [comment]
I'm getting the same issue, US - English, push to talk
Sing a song returns "i don't know how to help with that"

**Comment by salmanfarisvp** [comment]
@Fleker  i used English.

**Comment by Fleker** [comment]
Are you using pushtotalk or hotword?

**Comment by salmanfarisvp** [comment]
i used both 'pushtotalk' and 'hotword' .

**Comment by Fleker** [comment]
Can either of you try other queries? Does something like "what's my name" work?

**Comment by salmanfarisvp** [comment]
Yes, "what's my name" and "what is my location" is working fine .

**Comment by Fleker** [comment]
So you're only seeing a problem with that one command?

Can you try the `textinput` sample and see if anything is different?

**Comment by ameer1234567890** [comment]
I also have the same issue

**Comment by proppy** [comment]
Unable to reproduce with textinput from `HEAD`:
```
embedded-sdk-python 👺 env/bin/python google-assistant-sdk/googlesamples/assistant/grpc/textinput.py  --device-model-id foo --device-id foo
INFO:root:Connecting to embeddedassistant.googleapis.com
: what time is it
<you> what time is it
<@assistant> The time is 12:35 AM.
: sing a song
<you> sing a song
<@assistant> 🎤 I'm singing you a song because songs are great
And songs make you happy
This is a song for you
```

**Comment by salmanfarisvp** [comment]
@proppy i got error on both `pushtotalk` and `hotword`

**Comment by shavanni** [comment]
Hi 
This issue is still occurring on latest image 2018-08-03 using pushtotalk (assistant_grpc_demo.py).
